tests/unit-core/test-symbol.c

```diff
@@ -191,6 +191,130 @@ main (void)
   jerry_release_value (error_obj);
   jerry_release_value (null_value);
 
+  const jerry_char_t obj_src[] = ""
+  "({"
+  "  [Symbol.hasInstance]: 1,"
+  "  [Symbol.isConcatSpreadable]: 2,"
+  "  [Symbol.iterator]: 3,"
+  "  [Symbol.asyncIterator]: 4,"
+  "  [Symbol.match]: 5,"
+  "  [Symbol.replace]: 6,"
+  "  [Symbol.search]: 7,"
+  "  [Symbol.species]: 8,"
+  "  [Symbol.split]: 9,"
+  "  [Symbol.toPrimitive]: 10,"
+  "  [Symbol.toStringTag]: 11,"
+  "  [Symbol.unscopables]: 12,"
+  "})";
+
+  const char *symbols[] =
+  {
+    "hasInstance",
+    "isConcatSpreadable",
+    "iterator",
+    "asyncIterator",
+    "match",
+    "replace",
+    "search",
+    "species",
+    "split",
+    "toPrimitive",
+    "toStringTag",
+    "unscopables",
+  };
+
+  jerry_value_t obj = jerry_eval (obj_src, sizeof (obj_src) - 1, JERRY_PARSE_NO_OPTS);
+  TEST_ASSERT (jerry_value_is_object (obj));
+
+  jerry_value_t global_obj = jerry_get_global_object ();
+  jerry_value_t symbol_str = jerry_create_string ((const jerry_char_t *) "Symbol");
+  jerry_value_t builtin_symbol = jerry_get_property (global_obj, symbol_str);
+  TEST_ASSERT (jerry_value_is_object (builtin_symbol));
+
+  double expected = 1.0;
+  uint32_t prop_index = 0;
+
+  for (jerry_well_known_symbol_t id = JERRY_SYMBOL_HAS_INSTANCE;
+       id < JERRY_SYMBOL__NUMBER;
+       id++, expected++, prop_index++)
+  {
+    jerry_value_t well_known_symbol = jerry_get_well_known_symbol (id);
+
+    jerry_value_t prop_str = jerry_create_string ((const jerry_char_t *) symbols[prop_index]);
+    jerry_value_t current_global_symbol = jerry_get_property (builtin_symbol, prop_str);
+    jerry_release_value (prop_str);
+
+    jerry_value_t relation = jerry_binary_operation (JERRY_BIN_OP_STRICT_EQUAL,
+                                                     well_known_symbol,
+                                                     current_global_symbol);
+
+    TEST_ASSERT (jerry_value_is_boolean (relation)
+                 && jerry_get_boolean_value (relation));
+
+    jerry_release_value (relation);
+
+    jerry_value_t prop_result_wn = jerry_get_property (obj, well_known_symbol);
+    jerry_value_t prop_result_global = jerry_get_property (obj, well_known_symbol);
```

Comment:
These two `get_property` calls are doing the same thing (the inputs are the same). Is it intentional?

Reply:
Good catch, the second should have been `current_global_symbol`